test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/MessagePackHubProtocolTests.cs

```diff
@@ -35,6 +35,9 @@ public class MessagePackHubProtocolTests
             new object[] { new[] { new CompletionMessage("xyz", error: null, result: new CustomObject(), hasResult: true) } },
             new object[] { new[] { new CompletionMessage("xyz", error: null, result: new[] { new CustomObject(), new CustomObject() }, hasResult: true) } },
 
+            new object[] { new[] { new StreamCompletionMessage("xyz", error: null) } },
```

Comment:
@BrennanConroy  - I just noticed you did not add tests for CancellationMessage here

Reply:
Nevermind - I added it.